src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java

```diff
@@ -52,14 +55,20 @@ public void setUp() {
     @Test
     public void scenarios_should_behave_as_specified() throws IOException {
 
+        loadPropertiesIntoMapValueExpander();
+
+        for (Fixture fixture : fixtures) {
+            fixture.prepare();
+        }
+
+
         assertFalse(
             "Verifiers are configured",
             verifiers.isEmpty()
         );
 
-        loadPropertiesIntoMapValueExpander();
-
-        String scenarioPattern = System.getProperty("scenario");
+        //String scenarioPattern = System.getProperty("scenario");
+        String scenarioPattern = "RIA-941";
```

Comment:
I spotted this on my branch and have fixed it but not pushed it yet - was my bad.
Please remove this line and uncomment:
        `//String scenarioPattern = System.getProperty("scenario");`

Reply:
done